models/repo_mirror.go

```diff
@@ -264,6 +264,21 @@ func SyncMirrors() {
 			log.Error(4, "UpdateMirror [%s]: %v", repoID, err)
 			continue
 		}
+
```

Comment:
But line 263 has updated all the columns of the records.

Reply:
yes. It only update the `NextUpdateUnix` value. Right?

Reply:
but we need to update the `updated_unix` field of `repo` table.

Reply:
@appleboy. No. It should only update `NextUpdateUnix` maybe I don't know. But in fact it will update all the columns included `updated_unix`. So that your PR will update the record twice.

Reply:
@lunny this just update the `mirror` table and this PR update `updated_unix` field on the other table `repository`

Reply:
Oh, yes you are right. And maybe you can also refactor there to use one DB connection but not get connection from cache.

Reply:
@lunny Done. See https://github.com/go-gitea/gitea/pull/3551/commits/e5e87651732fadb31ca1be30c939f6d291f39416